openmc/mgxs/mgxs.py

```diff
@@ -824,9 +824,9 @@ def get_subdomain_avg_xs(self, subdomains='all'):
             mean = tally.get_reshaped_data(value='mean')
             std_dev = tally.get_reshaped_data(value='std_dev')
 
-            # Get the mean of the mean, std. dev. across requested subdomains
-            mean = np.mean(mean[subdomains, ...], axis=0)
-            std_dev = np.mean(std_dev[subdomains, ...]**2, axis=0)
+            # Get the mean, std. dev. across requested subdomains
+            mean = np.sum(mean[subdomains, ...], axis=0)
+            std_dev = np.sum(std_dev[subdomains, ...]**2, axis=0)
```

Comment:
Maybe my brain is not working because it's Monday morning.. isn't the only difference between `np.mean` and `np.sum` a factor of *N* being the length of the array?